br/pkg/lightning/restore/restore.go

```diff
@@ -1716,18 +1725,26 @@ func (rc *Controller) isLocalBackend() bool {
 // 4. Lightning configuration
 // before restore tables start.
 func (rc *Controller) preCheckRequirements(ctx context.Context) error {
-	if err := rc.ClusterIsAvailable(ctx); err != nil {
-		return errors.Trace(err)
-	}
+	if rc.cfg.App.CheckRequirements {
+		if err := rc.ClusterIsAvailable(ctx); err != nil {
```

Comment:
https://github.com/pingcap/tidb/blob/86b464b6471f15f918c2f10a400c7f6b93072d91/br/pkg/lightning/restore/check_info.go#L171 
this line can be removed